documentation/docs/core-concepts/anatomy-of-a-request.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Anatomy of a Request
+path: /anatomy-of-a-request/
+---
+
+# Anatomy of a Request
+
+In the previous sections, we learned about Fusion.js plugins and their ability to define middleware functions that can read the `request` object and/or make changes to it. In this section, we'll cover what happens when a request comes into the Fusion.js server, the order that plugins are accessed, and when rendering occurs.
+
+We'll be using the following example for this discussion.
+
+```js
+const app = new App();
+app.register(FirstPlugin);
+app.register(SecondPlugin);
+app.register(StandalonePlugin);
```

Comment:
It may be useful to either reorder these or mention that the order can be changed to whatever you want. People may assume (coming from an express world) that these are ordered as seen here, but the really drive it home we could mention that they could be in any order and it would still world correctly.

Reply:
👍 That's a good idea.